src/dstack/_internal/core/backends/base/compute.py

```diff
@@ -39,10 +64,20 @@ def run_job(
     ) -> LaunchedInstanceInfo:
         pass
 
+    @abstractmethod
```

Comment:
@abstractmethod should be removed since it forces all backends to implement create_instance().

Reply:
Thanks! I've removed abstractmethod.